src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1052 @@ metadata_pkg.(
+
+# Operations
+
+- `predict(mach, Xnew)`: Return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `coefficients`: The linear coefficients determined by the model.
+- `intercept`: The intercept determined by the model.
+
+# Examples
+
+```
+using MLJ
+using MLJBase: augment_X
+using DataFrames
+
+LinearRegressor = @load MultitargetLinearRegressor pkg=MultivariateStats
+RidgeRegressor = @load MultitargetRidgeRegressor pkg=MultivariateStats
+
+X = augment_X(randn(100, 80), true)
+θ = randn((81,4))
+y = X * θ
+X, y = map(x -> DataFrame(x, :auto), (X, y))
+
+# linear_regressor = LinearRegressor() # positive semi definite error for cholesky :(
+# mach = machine(linear_regressor, X, y) |> fit!
+# llsq_coef = fitted_params(mach).coefficients
+#
+# ridge_regressor = RidgeRegressor(lambda=0)
+# ridge_mach = machine(ridge_regressor, X, y) |> fit!
+# coef = fitted_params(ridge_mach).coefficients
+# difference = llsq_coef - coef
+# @info "difference between λ=0 ridge and llsq" mean(difference) std(difference)
+
+
+ridge_regressor = RidgeRegressor(lambda=1.5)
+ridge_mach = machine(ridge_regressor, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 60)
+yhat = predict(mach, Xnew) # new predictions
+```
+
+See also
+[`LinearRegressor`](@ref), [`MultitargetLinearRegressor`](@ref), [`RidgeRegressor`](@ref)
+"""
+MultitargetRidgeRegressor
+
+"""
+$(MMI.doc_header(PCA))
+
+`PCA` Principal component analysis. Learns a linear transformation to
+project the data  on a lower dimensional space while preserving most of the initial
+variance.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+# Hyper-parameters
+
+- `maxoutdim=0`: The maximum number of output dimensions. If not set, defaults to
+  0, where all components are kept (e.g., the number of components/output dimensions
+  is equal to the size of the smallest dimension of the training matrix)
+- `method=:auto`: The method to use to solve the problem. Choices are
+    - `:svd`: Support Vector Decomposition of the matrix.
+    - `:cov`: Covariance matrix decomposition.
+    - `:auto`: Use `:cov` if the matrices first dimension is smaller than its second dimension
+      otherwise use `:svd`
+- `pratio::Float64=0.99`: The ratio of variance preserved after the transformation
+- `mean=nothing`: if set to nothing(default) centering will be computed and applied,
+  if set to `0` no centering(assumed pre-centered), if a vector is passed,
+  the centering is done with that vector.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return lower dimensional projection of the target given new
+  features `Xnew` having the same Scitype as `X` above.
```

Comment:
I think the word "target" may cause confusion with use in supervised learning. Since this is a transformer, "input" might be a better word that "features". So how about:
```suggestion
- `transform(mach, Xnew)`: Return a lower dimensional projection of the input 
  `Xnew` having the same scitype as `X` above.
```